mqtt/mqtt-bridge/src/client.rs

```diff
@@ -313,15 +321,14 @@ where
         }
     }
 
-    pub async fn subscribe(&mut self, topics: &[String]) -> Result<(), ClientError> {
-        info!("subscribing to topics");
-        let subscriptions = topics.iter().map(|topic| proto::SubscribeTo {
-            topic_filter: topic.to_string(),
+    async fn subscribe(&mut self, topics: Vec<String>) -> Result<(), ClientError> {
```

Comment:
Why did you change to Vec? I was using Vec initially and I think clippy complained about using &[String]

Reply:
I refactored that to consume vec without cloning elements. That's why clippy is happy now :)